docs/en/notes/changelog.md

```diff
@@ -1,5 +1,50 @@
 # Changelog of v3.x
 
+## v3.0.0rc2 (4/11/2022)
+
```

Comment:
Add a line to indicate that we upgrade the version of MMEngine due to #9058, or we can update line 24.